recipe/bld.bat

```diff
@@ -1,4 +1,50 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat /XF Docs /XF Examples
+if errorlevel 1 exit 1
```

Comment:
Weird, `robocopy` still seems to be erroring, but it looks like it copied things ok. Any ideas @mbargull?

Reply:
Looks like your last commit isn't necessary as the list of excluded files is identical:
https://ci.appveyor.com/project/conda-forge/nsis-feedstock/build/1.0.30/job/poflow3okmx0b9jb#L521
https://ci.appveyor.com/project/conda-forge/nsis-feedstock/build/1.0.31/job/yx2gtiije3oj2ku8#L521

As for the error: `robocopy` shows that it skipped a file:
https://ci.appveyor.com/project/conda-forge/nsis-feedstock/build/1.0.31/job/yx2gtiije3oj2ku8#L1030
But it doesn't state which file is skipped -- how helpful! -_-
Maybe some file with a system attribute or something? Don't know.. Can you try
```bat
robocopy . "%PREFIX_NSIS%" /V /S /XF bld.bat Docs Examples
```
? Acc. to https://technet.microsoft.com/en-us/library/cc733145(v=ws.10).aspx `/v` also shows the skipped files.

Reply:
Yeah that was the impression I got as well.

Ah ok. Have add the `/V` flag. Let's see what happens.

Reply:
Does that help?

Reply:
Ok, the skipped file is just the `bld.bat` that was meant to be skipped...

Reply:
Bwahh: https://ss64.com/nt/robocopy-exit.html
> Any value greater than 7 indicates that there was at least one failure during the copy operation.

Reply:
So we might need `errorlevel 8` then?

Reply:
But then we might need to reset the error level afterwards, e.g., something like
```bat
robocopy ...
if errorlevel 8 exit 1
call :reset_error_level
...
REM at the end of bld.bat:
exit 0
:reset_error_level
exit /B 0
```

Reply:
Was starting to wondering if `robocopy` was just giving screwy errorlevels. Surprised we are only running into it now. Pushed a commit, which should help. Could you please double check if that looks ok?

Reply:
Side note: Why isn't skipping the other directories we told it to skip? Do we need the path instead of the directory name?

Reply:
Ah, you may be right: (https://ss64.com/nt/robocopy-exit.html)
> /xd <Directory>[ ...] | Excludes directories that match the specified names and paths.

So
```bat
robocopy . "%PREFIX_NSIS%" /S /XF bld.bat /XD Docs Examples
```
should do the trick, I think.

Reply:
Thanks. Added. Let's see how it goes.